src/mono/wasm/runtime/crypto-worker.ts

```diff
@@ -358,7 +359,16 @@ class LibraryChannel {
         return String.fromCharCode.apply(null, slicedMessage);
     }
 
-    private acquire_lock() {
+    private using_lock(cb: Function) {
```

Comment:
I'm not a web dev, but is `cb` a "typical" name for callback? Can it be named `callback` or `action`? That seems to be more readable.